aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -147,7 +147,8 @@ impl AptosVM {
     }
 
     pub fn set_num_shards_once(mut num_shards: usize) {
-        num_shards = min(num_shards, 1);
+        num_shards = max(num_shards, 1);
```

Comment:
Let's add a comment here, because if it wouldn't work for general transactions at the moment then it's critical to explain here.

Reply:
Done.